book/src/nextest.md

```diff
@@ -0,0 +1,27 @@
+# cargo-mutants with nextest
```

Comment:
@sunshowers could you have a look over the doc page and point out anything inaccurate?